cli-artemis-jms/src/main/java/com/redhat/mqe/acc/AccSenderClient.java

```diff
@@ -24,7 +24,7 @@
 import com.redhat.mqe.lib.JmsMessageFormatter;
 import com.redhat.mqe.lib.Utils;
 import com.redhat.mqe.lib.message.MessageProvider;
-import org.apache.activemq.artemis.api.core.ActiveMQUnBlockedException;
+import org.apache.activemq.artemis.api.core.*;
```

Comment:
When do we want to use * import? By default, IntelliJ will put it there if you have at least 3 classes imported from the same package.

I remember that once we had decision always name the imported classes explicitly.

Maybe increase the setting in Intellij, from 3 to 5, 10?